<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainChatWindow</class>
 <widget class="QMainWindow" name="MainChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chat Application</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <widget class="QWidget" name="leftSection" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
       QWidget#leftSection {
           background-color: #f0f0f0;
           border-right: 1px solid #dcdcdc;
       }
       QListWidget#chatList {
           font-size: 14px;
           padding: 5px;
       }
      </string>
      </property>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLineEdit" name="userSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Search by Username...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="chatList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightSection" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
       QWidget#rightSection {
           background-color: #ffffff;
       }
       QWidget#header {
           background-color: #f8f8f8;
           border-bottom: 1px solid #dcdcdc;
           padding: 10px;
       }
       QLabel#contactName {
           font-size: 18px;
           font-weight: bold;
			color:black;
       }
       QListWidget#conversationHistory {
           padding: 10px;
       }
       QWidget#composer {
           background-color: #f8f8f8;
           border-top: 1px solid #dcdcdc;
           padding: 5px;
       }
       QLineEdit#messageInput {
           font-size: 14px;
           padding: 5px;
           border: 1px solid #dcdcdc;
           border-radius: 5px;
       }
       QPushButton#attachButton, QPushButton#sendButton {
           font-size: 14px;
           padding: 5px 10px;
           margin-left: 5px;
       }
       QPushButton#sendButton {
           background-color: #4CAF50;
           color: white;
           border: none;
           border-radius: 5px;
       }
       QPushButton#sendButton:hover {
           background-color: #45a049;
       }
  QPushButton#attachButton:hover {
           background-color: #FFFFFF;
color:black;
		  border: none;
           border-radius: 5px;
       }
      </string>
      </property>
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QWidget" name="header" native="true">
         <layout class="QHBoxLayout" name="headerLayout" stretch="0">
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="contactName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>4</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Ink Free</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>Open a Chat, let's Chat</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="conversationHistory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="composer" native="true">
         <layout class="QHBoxLayout" name="composerLayout">
          <item>
           <widget class="QLineEdit" name="messageInput">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Type a message...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="attachButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Attach</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
